docs/application/dotnet/guides/nui/widget.md

```diff
@@ -0,0 +1,257 @@
+# NUI Widget Application
+
+[NUIWidgetApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html)  is to show a small version of application on home screen.
```

Comment:
> Suggestion: Please provide some more information in the introduction such as why do we use this, what is the purpose, when do we use, and so on.
> 
> For example,
> You can use [NUIWidgetApplication](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.NUIWidgetApplication.html) to create Tizen.NET widget applications, which are small versions of Tizen .NET applications on the home screen. You can launch these applications on the home screen when you select them in the **Add Widget** menu.
> 
> Please verify. This is just an example. Kindly elaborate the introduction.

"You can launch these applications on the home screen when you select them in the Add Widget menu." is not suitable for this guide because your rephase assumes "Add Widget" menu on the home screen.

I think the feature or definition of this module is better than "you can...", as the start wording.

Reply:
@journey2w, thank you for the correction.
Could we start with:

NUIWidgetApplication helps you to create Tizen.NET widget applications, which are .....

Secondly, request the author to please provide an introduction.